### ISSUE TITLE
[FEAT][Vision] allow use of sklearn metrics for classification

### ISSUE THREAD

**Issue by Nadav-Barak** [deepchecks/deepchecks#1635]
**Is your feature request related to a problem? Please describe.**
Ignite have a relatively small number of metric in oppose to sklearn. In addition a uniform API between tabular and classification metrics API is a big plus.

**Describe the solution you'd like**
Update vision class performance for vision classification to support the tabular metrics for classification.
i.e after digesting all batches and collection true labels, probabilities and predictions invoke the relevant sklearn metrics API in the same manner it is done in the tabular performance report.

**Comment by Nadav-Barak** [comment]
This will require a medium refactor in the vision class performance check
